Formula/cfitsio.rb

```diff
@@ -39,4 +40,18 @@ def install
       end
     end
   end
+
+  test do
+    cp Dir["#{pkgshare}/testprog/testprog*"], testpath
+    flags = (ENV.cflags || "").split + (ENV.cppflags || "").split + (ENV.ldflags || "").split
```

Comment:
@MikeMcQuaid Sure. Should I put everything in one commit or use one commit per formula?